README.md

```diff
@@ -44,7 +44,7 @@ Follow these directions to enable query linting during local development so your
 
 ### Environment Variables
 
-[env-cmd](https://github.com/toddbluhm/env-cmd#readme) is used to configure build environments for production, staging and development. This file is git ignored because it contains API keys that we do not want to publish. It should be named `.cmdrc.json` and placed in the `env/` folder at the root of the project. This file is required to deploy Spruce to production and to staging. Ask a team member to send you their copy of the file, which should look like the following:
+[env-cmd](https://github.com/toddbluhm/env-cmd#readme) is used to configure build environments for production, staging and development. We use two files to represent these various environments: `.cmdrc-local.json` for local builds with non-sensitive information, and `.cmdrc.json` for builds deployed to S3. This file is git ignored because it contains API keys that we do not want to publish. It should be named `.cmdrc.json` and placed in the `env/` folder at the root of the project. This file is required to deploy Spruce to production and to staging. Ask a team member to send you their copy of the file, which should look like the following:
```

Comment:
This change makes sense to me although I would advocate for splitting it out of this pr, to make reverts easier. (We can revert the feature without having to worry about reverting how we handle credentials) 
Also could we do this on parsley for consistency?

Reply:
https://github.com/evergreen-ci/spruce/pull/1686